CHANGELOG.md

```diff
@@ -30,7 +31,7 @@ n/a
   `Message::getName`.
 - `RetrySpec::retryDelay` method added to allow a message to be delayed when
   retrying, if the driver supports it.
-- `Driver::retry` now accepts an `int $delay` to support delayed retries. Not all
+- `Envelope::retry` now accepts an `int $delay` to support delayed retries. Not all
```

Comment:
Shouldn't this be part of the "Driver should no longer call" part? It's not really a fix.